exporters/variables-scss/src/helpers/specialCaseHelper.ts

```diff
@@ -0,0 +1,16 @@
+// if you want to add exceptions for specific tokens, add them here
+const specialCaseMap = new Map<string, string | number>([['breakpoint-mobile', 0]]);
+
+export const handleSpecialCase = <T extends string | number>(name: string, value: T): T => {
+  const specialCaseValue = specialCaseMap.get(name);
+
+  if (typeof specialCaseValue === 'number' && typeof value === 'number') {
+    return specialCaseValue as T;
+  }
+
+  if (typeof specialCaseValue === 'string' && typeof value === 'string') {
+    return specialCaseValue as T;
+  }
```

Comment:
These two conditions are returning the same result. Wouldn't it be better to combine them?

```
if (
  (typeof value === 'number' && typeof specialCaseValue === 'number') ||
  (typeof value === 'string' && typeof specialCaseValue === 'string')
) {
  return specialCaseValue as T;
}
  ```
  
  or maybe even:
  
  ```
if (typeof specialCaseValue === typeof value) {
    return specialCaseValue as T;
}
```